maat-court-data-api/src/main/java/gov/uk/courtdata/repository/FdcContributionsRepository.java

```diff
@@ -13,4 +14,7 @@ public interface FdcContributionsRepository extends JpaRepository<FdcContributio
     List<FdcContributionsEntity> findByStatus(FdcContributionsStatus status);
     List<FdcContributionsEntity> findByIdIn(Set<Integer> ids);
 
+    @Query(value = "SELECT TOGDATA.REF_DATA_MANAGEMENT.GET_CONFIG_PARAMETER('FDC_CALCULATION_DELAY', sysdate) FROM DUAL", nativeQuery = true)
```

Comment:
Simple statement to get access to the specific config that is desired.
Was no need for a more generic procedure.